spec/controllers/admin/features_controller_spec.rb

```diff
@@ -16,6 +15,69 @@
       get :show, :id => feature.id
       expect(response).to be_success
     end
-  end  
+  end
+
+  describe "#destroy" do
```

Comment:
This is really well-tested.  :+1: I'm not sure if this sort of integration testing should be done at the controller level - we have a few high-level integration specs in `spec/features`, and I'd like to do more at that level.  But I'm not saying you need to move this stuff now - just having it anywhere is great.